torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -719,21 +718,7 @@ def rewrite_node(node):
                 else:
                     dim0 = args[1]
                     dim1 = args[2]
-                permutation = list(range(rank))
-                permutation[dim0], permutation[dim1] = (
-                    permutation[dim1],
-                    permutation[dim0],
-                )
-                # TODO(#377): ttnn.permute fails when swapping inner-most dim = 1 for 2D
-                if rank == 2 and output_size[0] == 1:
-                    return None
```

Comment:
This might still fail. <2D tensors when converted back to torch might get 2D. The team is working on this right now.

Reply:
It is a no-op to transpose a < 2D tensor. We already have this shortcut for `aten.t` before and I extend it for `aten.transpose` too. So we can avoid this problem here